### ISSUE TITLE
Change `// test_output! output` to `//! test_output(output)`

### ISSUE THREAD

**Issue by voltrevo** [voltrevo/ValueScript#7]
`//! test_output(output)` would be clearer than `// test_output! output`. It's also clearer how it should generalize to other annotations. (This is not meant to be part of the language, just our internal tooling.)

For example, `tests/passing/anonymousMain.ts` is currently:

```ts
// test_output! "Done"

export default function () {
  return "Done";
}
```

This should be changed to:

```ts
//! test_output("Done")

export default function () {
  return "Done";
}
```

To be clear, this issue is also about changing `test_inputs.rs` so that the new annotation works. (Remove the old one too, don't support both.).

**Comment by SamantaTarun** [comment]
I would like to work on this issue

**Comment by voltrevo** [comment]
@tarunsamanta2k20 awesome! Do you need any more details to get started?

**Comment by voltrevo** [comment]
@tarunsamanta2k20 Still looking forward to your contribution here. Are you still intending to do this?

**Comment by voltrevo** [comment]
Closed by #11.
